src/main/java/org/folio/rest/impl/util/RequestsApiUtil.java

```diff
@@ -40,4 +41,32 @@ public static Errors samePositionInQueueError(String item, Integer position) {
 
     return errors;
   }
+
+  public static Errors requestStructureIsValid(Request request){
+    Request.RequestLevel requestLevel = request.getRequestLevel();
+    List<Error> errorList = new ArrayList<>();
+    if (requestLevel == Request.RequestLevel.ITEM) {
```

Comment:
Extract `request.getItemId() == null` and `request.getHoldingsRecordId() == null` into variables

Reply:
Resolved